app/clients/sms/twilio.py

```diff
@@ -48,8 +48,10 @@ def send_sms(self, to, content, reference, **kwargs) -> str:
 
         start_time = monotonic()
         callback_url = ""
-        if self._callback_notify_url_host:
-            callback_url = f"{self._callback_notify_url_host}/notifications/sms/twilio/{reference}"
+        # TODO: Following two lines are commented out because the callback url points to an internal url.
+        # TODO (cont): When reverse proxy endpoint is created, we can assign that to callback_url and uncomment.
+        # if self._callback_notify_url_host:
+        #    callback_url = f"{self._callback_notify_url_host}/notifications/sms/twilio/{reference}"
```

Comment:
Do we have a ticket for this proxy work? If so, can the ticket # please be added to this TODO comment? If not: why not? Do we ever plan to implement a reverse proxy? If so, it should have a ticket and be added here.

Reply:
good call out.  I created a [ticket](https://github.com/department-of-veterans-affairs/vanotify-team/issues/716) for the reverse proxy last week, I will add that to the ToDo.